### DISCUSSION TITLE
Using .js libraries to extend features

### DISCUSSION THREAD

**Discussion by cjohnsonuk** [hluk/CopyQ#2029] [Ideas]
Is it possible to use .js libraries such as the image editing ones here https://flatlogic.com/blog/12-javascript-image-manipulation-libraries-for-your-next-web-app/ such as Jimp , fabric.js etc in the scripts part of CopyQ?

I'd like to be able to add  menu options to CopyQ to resize, add icons, borders, watermarks etc to my copied images without having to break open an image editor every time.

**Comment by hluk** [comment]
You would need to test it but the scripting language in CopyQ does **not support many the features in a recent versions of JavaScript**.
